initiatives/2020-04-docs.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "type": "sourcecred/initiativeFile",
+    "version": "0.1.0"
+  },
+  {
+    "title": "Initial Documentation Push",
+    "timestampIso": "2020-04-01T10:00:00.000Z",
+    "weight": {
+      "incomplete": 200,
+      "complete": 1000
+    },
+    "completed": false,
+    "champions": [
+    ],
+    "dependencies": [
+    ],
+    "references": [
+      "https://github.com/sourcecred/docs/issues/5",
+      "https://discourse.sourcecred.io/t/creating-documentation-on-github-review-culture/582",
+      "https://discourse.sourcecred.io/t/documentation-glossary-and-wishlist/354",
+      "https://discourse.sourcecred.io/t/weekly-community-call-1-28-2020/605",
+      "https://discourse.sourcecred.io/t/weekly-community-call-1-14-2019/578",
+      "https://discourse.sourcecred.io/t/weekly-community-call-1-7-2020/572"
+    ],
+    "contributions": [
+      "https://github.com/sourcecred/docs/47",
```

Comment:
Also getting the 404s. It looks like the URL is missing the `pull/` part of the path. E.g. presumably this one is:
`https://github.com/sourcecred/docs/pull/47`